src/chains/algorand.ts

```diff
@@ -0,0 +1,74 @@
+const algosdk = require('algosdk');
+const nacl = require('tweetnacl');
+
+function privateKeyToArray(privateKey: string) {
+    return Buffer.from(privateKey.substring(2,privateKey.length), 'hex').toJSON().data
+}
+
+function keyPairFromSecretKey(sk: Uint8Array) {
+    return nacl.sign.keyPair.fromSecretKey(sk);
+  }
+
+export default class utils {
+
+    static createWallet(): object {
+        const wallet = algosdk.generateAccount();
+        const mnemonic = algosdk.secretKeyToMnemonic(wallet.sk);
+        const privateKey = '0x' + Buffer.from(wallet.sk).toString('hex')
+        const keypair = keyPairFromSecretKey(wallet.sk)
+        const publicKey = '0x' + Buffer.from(keypair.publicKey).toString('hex')
+
+        return {
+            mnemonic,
+            privateKey,
+            publicKey,
+            address: wallet.addr
+        }
+    }
+
+    static getWallet(mnemonic: string): object {
+        const wallet = algosdk.mnemonicToSecretKey(mnemonic)
+        const privateKey = '0x' + Buffer.from(wallet.sk).toString('hex')
+        const keypair = keyPairFromSecretKey(wallet.sk)
+        const publicKey = '0x' + Buffer.from(keypair.publicKey).toString('hex')
```

Comment:
@saadibrahim I'm pretty sure Algorand public keys / addresses aren't represented as `0x` prefixed hex strings.

Reply:
yep, I was trynna follow the pattern of the lib, there is no use for public key tho from what I can see. Is ti okay to just make this the address?

Reply:
You'll need to look at the Algorand documentation to see what the public key should be.

The public key is important for other use cases of the library.

Reply:
Oh, just saw your next comment. If public key = address, then yes, make them the same.

Reply:
from their documentation:

<img width="975" alt="Screen Shot 2021-12-06 at 4 39 08 PM" src="https://user-images.githubusercontent.com/4365774/144851780-2e82eda5-31c4-4c56-be16-f81e377512be.png">

It implies the public key is pretty much of no use to the end user, and I do not see how we would be using the public key, the mnenomic, secret/private key and the address is all we need.

Reply:
If it's easy to get the public key using the AlgoSDK, I would still like to return it for completeness.

Reply:
It isnt, they dont provide a method. I dove into the sdk code and managed to generate it using nacl but I realized it was unnecessary code for no reason, there was no use for that public key.